### DISCUSSION TITLE
ingredient unit conversion from volumetric measurements to weight measurements

### DISCUSSION THREAD

**Discussion by FuzzyWereBeast** [TandoorRecipes/recipes#2510] [Ideas]
With the addition of conversions, will we have the ability to select how we see the ingredients?

Most of my family is content using volumetric measurements where I prefer to use weights.  Could we have a slider or button that would allow conversions should one exist.

example.
recipe calls for 2 cups of flour.  Since flour is 30g for 1/4 cup, it would read 2 cups flour if volumetric measurements are selected or 240g if weights are selected.

Also, is the system going to realize that 4 tbsp = 1/4 cup or will we need to add a conversion for each type of volumetric measurement?

**Comment by vabene1111** [comment]
all conversions happen automatically and i am planning on allowing some kind of preferred unit or a setting to define the unit preferences. Not something coming directly as I want this feature to mature a bit but at some point.

**Comment by Hugo-Persson** [comment]
Is this something that is planned? Would be great to automatically convert flour in volume to weight by some preference
